airflow/www/views.py

```diff
@@ -5177,6 +5177,7 @@ class DagRunModelView(AirflowPrivilegeVerifierModelView):
         "queued_at",
         "start_date",
         "end_date",
+        # Ordering is not supported on association proxies
         # "note", # todo: maybe figure out how to re-enable this
```

Comment:
The new comment is arguably unrelated and should be removed from this PR.

Reply:
Thank you! Removed it